src/main/ruby/core/rbconfig.rb

```diff
@@ -47,6 +47,7 @@ module RbConfig
   ruby_version   = ruby_api_version
 
   arch         = "#{host_cpu}-#{host_os}"
+  host         = "#{host_cpu}-unknown-#{host_os}-gnu"
```

Comment:
What does `unknown` and `gnu` mean?

Reply:
Comment has now been added explaining target triple structure somewhat.